test/js-api/jsapi.js

```diff
@@ -881,4 +881,57 @@ test(() => {
   assert_equals(table.get(2)(), 46);
 }, "Tables export cached");
 
+const exportingModuleLongIdentityFn = (() => {
+    let builder = new WasmModuleBuilder();
+
+    builder
+        .addFunction('id', kSig_l_l)
+        .addBody([ kExprGetLocal, 0 ])
+        .exportFunc();
+
+    return builder.toBuffer();
+})();
+
+test(() => {
+  let module = new WebAssembly.Module(exportingModuleLongIdentityFn);
+  let instance = new WebAssembly.Instance(module);
+
+  let value = 2n ** 63n;
```

Comment:
That assumes the host has the BigInt proposal support?

Reply:
Yes, it does. When we [split up this file](https://github.com/WebAssembly/spec/issues/596), this test should probably be split off to let the others keep running on systems that don't support BigInt.